.browserslistrc

```diff
@@ -0,0 +1,2 @@
+last 2 versions, > 2%, not ie <= 11, not dead, not opera > 0, not op_mini all, not op_mob > 0
```

Comment:
Formatting the list + adding the keyword `major`.

```suggestion
last 2 major versions, 
> 2%, 
not dead, 
not ie > 0, 
not opera > 0, 
not op_mini all, 
not op_mob > 0,
```

The result is:
```
and_chr 93
and_ff 92
and_qq 10.4
and_uc 12.12
android 93
baidu 7.12
chrome 93
chrome 92
edge 93
edge 92
firefox 92
firefox 91
ios_saf 14.5-14.7
ios_saf 14.0-14.4
ios_saf 13.4-13.7
ios_saf 13.3
ios_saf 13.2
ios_saf 13.0-13.1
kaios 2.5
safari 14.1
safari 14
safari 13.1
safari 13
samsung 14.0
samsung 13.0
```

Reply:
I'm not opposed at all. @himerus what do you think?